airflow/www_rbac/.eslintignore

```diff
@@ -6,3 +6,8 @@ jqClock.min.js
 coverage/**
 static/dist/*
 static/docs/*
+templates/airflow/dags.html
```

Comment:
These are the files which includes all the custom javascript in the codebase, and which need to be lint properly. We should not include them in `.eslintignore` at all and try to fix the issue. (or look into the eslint jinja template plugins)